{"uid":"696b778234827e03","name":"test_validate_payment[api-19]","fullName":"tests.endpoints.test_payments#test_validate_payment","historyId":"5dad880742ddfd39d1015f29f76581bb","time":{"start":1733146279457,"stop":1733146280397,"duration":940},"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nusers\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing\nmeta\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing","statusTrace":"api_client = <api_handler.api_client.ApiClient object at 0x7ff4a048d340>\nx_task_value = 'api-19'\n\n    @allure.feature(\"Orders API\")\n    @allure.story(\"Duplicate item in order\")\n    @pytest.mark.parametrize(\"x_task_value\", [\"api-19\"])\n    def test_validate_payment(api_client, x_task_value):\n        users_response = api_client.get_users(x_task_value, 0)\n>       users_dto = Users(**users_response.json())\nE       pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nE       users\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\nE       meta\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\n\ntests/endpoints/test_payments.py:14: ValidationError","flaky":true,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1733146274602,"stop":1733146274603,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"api_client","time":{"start":1733146274603,"stop":1733146274603,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1733146274603,"stop":1733146274603,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup_test","time":{"start":1733146274604,"stop":1733146276261,"duration":1657},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"worker_id","time":{"start":1733146274604,"stop":1733146274604,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nusers\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing\nmeta\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing","statusTrace":"api_client = <api_handler.api_client.ApiClient object at 0x7ff4a048d340>\nx_task_value = 'api-19'\n\n    @allure.feature(\"Orders API\")\n    @allure.story(\"Duplicate item in order\")\n    @pytest.mark.parametrize(\"x_task_value\", [\"api-19\"])\n    def test_validate_payment(api_client, x_task_value):\n        users_response = api_client.get_users(x_task_value, 0)\n>       users_dto = Users(**users_response.json())\nE       pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nE       users\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\nE       meta\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\n\ntests/endpoints/test_payments.py:14: ValidationError","steps":[],"attachments":[{"uid":"fcd213b5f9156a28","name":"log","source":"fcd213b5f9156a28.txt","type":"text/plain","size":320},{"uid":"ee18b08c29196b82","name":"stderr","source":"ee18b08c29196b82.txt","type":"text/plain","size":314}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"setup_test::0","time":{"start":1733146280400,"stop":1733146280400,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"story","value":"Duplicate item in order"},{"name":"feature","value":"Orders API"},{"name":"parentSuite","value":"tests.endpoints"},{"name":"suite","value":"test_payments"},{"name":"host","value":"fv-az888-715"},{"name":"thread","value":"2805-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.endpoints.test_payments"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"x_task_value","value":"'api-19'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":3,"broken":2,"skipped":0,"passed":3,"unknown":0,"total":8},"items":[{"uid":"8ed89326eb4ba61e","reportUrl":"https://ils-808.github.io/qahack/100//#testresult/8ed89326eb4ba61e","status":"failed","statusDetails":"AssertionError: Response validation failed: 2 validation errors for PaymentResponse\n  created_at\n    Field required [type=missing, input_value={'amount': 5999, 'order_u...4dd0-ae71-ec07cf3d9260'}, input_type=dict]\n      For further information visit https://errors.pydantic.dev/2.9/v/missing\n  updated_at\n    Field required [type=missing, input_value={'amount': 5999, 'order_u...4dd0-ae71-ec07cf3d9260'}, input_type=dict]\n      For further information visit https://errors.pydantic.dev/2.9/v/missing\nassert False","time":{"start":1733146069713,"stop":1733146074602,"duration":4889}},{"uid":"eec15e993a078640","reportUrl":"https://ils-808.github.io/qahack/99//#testresult/eec15e993a078640","status":"broken","statusDetails":"pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nusers\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing\nmeta\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing","time":{"start":1733145945341,"stop":1733145946236,"duration":895}},{"uid":"1c90389699cd308b","reportUrl":"https://ils-808.github.io/qahack/98//#testresult/1c90389699cd308b","status":"passed","time":{"start":1733145788748,"stop":1733145792399,"duration":3651}},{"uid":"7a52d4108ba5d015","reportUrl":"https://ils-808.github.io/qahack/63//#testresult/7a52d4108ba5d015","status":"passed","time":{"start":1733086115026,"stop":1733086118578,"duration":3552}},{"uid":"ee798a145046cc4f","reportUrl":"https://ils-808.github.io/qahack/62//#testresult/ee798a145046cc4f","status":"failed","statusDetails":"AssertionError: Response validation failed: 2 validation errors for PaymentResponse\n  created_at\n    Field required [type=missing, input_value={'amount': 5999, 'order_u...4265-828d-6509050a18e8'}, input_type=dict]\n      For further information visit https://errors.pydantic.dev/2.9/v/missing\n  updated_at\n    Field required [type=missing, input_value={'amount': 5999, 'order_u...4265-828d-6509050a18e8'}, input_type=dict]\n      For further information visit https://errors.pydantic.dev/2.9/v/missing\nassert False","time":{"start":1733084995786,"stop":1733084999236,"duration":3450}},{"uid":"6ed1ab9322b0b8f8","reportUrl":"https://ils-808.github.io/qahack/61//#testresult/6ed1ab9322b0b8f8","status":"failed","statusDetails":"AssertionError: Response validation failed: 2 validation errors for PaymentResponse\n  created_at\n    Field required [type=missing, input_value={'amount': 5999, 'order_u...4b6a-b741-61e0f653a32b'}, input_type=dict]\n      For further information visit https://errors.pydantic.dev/2.9/v/missing\n  updated_at\n    Field required [type=missing, input_value={'amount': 5999, 'order_u...4b6a-b741-61e0f653a32b'}, input_type=dict]\n      For further information visit https://errors.pydantic.dev/2.9/v/missing\nassert False","time":{"start":1733084774245,"stop":1733084778068,"duration":3823}},{"uid":"1cdc5125ef3203b2","reportUrl":"https://ils-808.github.io/qahack/60//#testresult/1cdc5125ef3203b2","status":"passed","time":{"start":1733084495967,"stop":1733084500412,"duration":4445}}]},"tags":[]},"source":"696b778234827e03.json","parameterValues":["'api-19'"]}